.github/workflows/prerelease.yml

```diff
@@ -52,18 +57,18 @@ jobs:
   #     - name: Setup Java JDK
   #       uses: actions/setup-java@v1.4.3
   #       with:
-  #         java-version: 11 
+  #         java-version: 11
 
   #     - name: Set Up CommandBox
   #       uses: elpete/setup-commandbox@v1.0.0
-      
+
   #     - name: Install and Configure Semantic Release
   #       run: |
   #         box install commandbox-semantic-release
```

Comment:
While we're changing the build script, can we pin to a major version range? `^2.0.0`?

Reply:
I don't think you can do that with the setup packages